runtime/integ_test.go

```diff
@@ -140,3 +145,159 @@ func runTask(ctx context.Context, c containerd.Container) (*commandResult, error
 		return nil, errors.New("context cancelled")
 	}
 }
+
+// setupBareMetalTest will run firecracker-containerd, the runtime shim, and
+// snapshotter. This will return the cleanup function needed to kill each
```

Comment:
The snapshotter is part of containerd at this point and shouldn't be run separately.  We've removed the external snapshotter implementation from this repository.